domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -147,10 +155,11 @@ private static String randomUuid() {
   }
 
   /**
-   * Return a handler that, when invoked, asserts that the AsyncResult succeeded, the Response has httpStatus,
-   * and the toString() of the entity of Response contains the snippet.
+   * Return a handler that, when invoked, asserts that the AsyncResult succeeded, the Response has
+   * httpStatus, and the toString() of the entity of Response contains the snippet.
    */
-  private Handler<AsyncResult<Response>> asyncAssertSuccess(TestContext testContext, int httpStatus, String snippet) {
+  private Handler<AsyncResult<Response>> asyncAssertSuccess(TestContext testContext, int httpStatus,
+    String snippet) {
```

Comment:
After a line break within a method's parameter four additional spaces should be added to the indentation.
Using two is misleading because it looks as if it were starting the actual code block.
Can you adjust your IDE's settings and fix all those wrapped lines?